src/app/store/config/visualClues.ts

```diff
@@ -24,4 +24,7 @@ export const visualClues = defineVisualClues({
   CompleteCollectorProfileMessage: {
     description: "The message shown if the collector profile is incomplete",
   },
+  AddedArtworkHasNoInsightsMessage_MyCTab: {
```

Comment:
What do you think about changing it to:

```suggestion
  AddArtworkWithoutInsightsMessage: {
```

?

Reply:
`AddArtworkWithoutInsightsMessage_MyCTab` may be?
We will have three more clues, four in total
Two of them for Insights tab and two for MyCollection

Reply:
Sounds good! Will they all be marked as seen independently or will they disappear together after the user has seen one of them?

Reply:
Good question
I would say they should disappear independently.
I'll be working on all these messages next week and see how they appear in different scenarios